index.html

```diff
@@ -19,42 +20,47 @@
 <header id="top">
     <nav>
         <ul>
-            <p><a herf=index.html>Home</a></p>
-            <p><a herf=links.html>Links</a></p>
-            <p><a herf=glossary.html>glossary</a></p>
-            </ul>
-            </nav> </header>
-            <main> </main>
-            <footer> </footer>
-            ◤<a href="Index.html"><title>Back To Top</title></a>       
```

Comment:
Some strange placement of elements here... use a validator to catch these things!